app/src/main/java/org/simple/clinic/storage/migrations/Migration_73.kt

```diff
@@ -0,0 +1,17 @@
+package org.simple.clinic.storage.migrations
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+import org.simple.clinic.storage.inTransaction
+import javax.inject.Inject
+
+@Suppress("Classname")
+class Migration_73 @Inject constructor() : Migration(72, 73) {
```

Comment:
Just an FYI, since we are squshing the commits when merging you can do the migration in a different commit so that it will be easier to review.